.github/workflows/github-token-rotation-prod.yml

```
@@ -0,0 +1,72 @@
+          fi
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          # Assume IAM role for GitHub Actions to access AWS resources
+          role-to-assume: ${{ secrets.GITHUB_TOKEN_ROTATION_ROLE_TO_ASSUME_PROD}}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Install PyJWT
+        run: |
+          pip3 install PyJWT      
```

yamllint

[error] trailing spaces

(trailing-spaces)